### PR TITLE
Update a few flag help strings for consistency and clarity

### PR DESCRIPTION
**Pull request by tianon**
While looking at the help for `-graph-driver` (`-graph-driver="": For docker to use a specific graph driver`) and realizing that it was probably supposed to be "Force", I noticed that our flag help strings were a little inconsistent in a few places (both in verbiage and in punctuation - some with periods, some without).  I'm still not 100% happy with what I came up with for `-H` after iterating through quite a few different wordings there, but at least it's more clear than what we've got now.

Some wordings, like the help text for `-iptables` were additionally misleading.  "Disable iptables within docker" could easily be misinterpreted to mean that docker would force disable existing iptables rules, which isn't a feeling we want to even hint at, because we're actually really, really careful not to break anything that already exists. :)

Here's the full before and after output:

```
$ docker -h # before
  -D=false: Debug mode
  -H=[unix:///var/run/docker.sock]: tcp://host:port to bind/connect to or unix://path/to/socket to use
  -api-enable-cors=false: Enable CORS requests in the remote api.
  -b="": Attach containers to a pre-existing network bridge. Use 'none' to disable container networking
  -d=false: Daemon mode
  -dns="": Set custom dns servers
  -g="/var/lib/docker": Path to use as the root of the docker runtime.
  -graph-driver="": For docker to use a specific graph driver
  -icc=true: Enable inter-container communication
  -ip="0.0.0.0": Default ip address to use when binding a containers ports
  -iptables=true: Disable iptables within docker
  -p="/var/run/docker.pid": File containing process PID
  -r=true: Restart previously running containers
  -v=false: Print version information and quit

$ docker -h # after
  -D=false: Enable debug mode
  -H=[unix:///var/run/docker.sock]: Multiple tcp://host:port or unix://path/to/socket to bind in daemon mode, single connection otherwise
  -api-enable-cors=false: Enable CORS headers in the remote API
  -b="": Attach containers to a pre-existing network bridge; use 'none' to disable container networking
  -d=false: Eanble daemon mode
  -dns="": Force docker to use specific DNS servers
  -g="/var/lib/docker": Path to use as the root of the docker runtime
  -graph-driver="": Force docker runtime to use a specific graph driver
  -icc=true: Enable inter-container communication
  -ip="0.0.0.0": Default IP address to use when binding container ports
  -iptables=true: Disable docker's addition of iptables rules
  -p="/var/run/docker.pid": Path to use for daemon PID file
  -r=true: Restart previously running containers
  -v=false: Print version information and quit
```

Of course, I'll be happy to update this more if anyone has better suggestions for wording. :)

### CODE REVIEW

**Comment by crosbymichael** [comment]
LGTM

File: docker/docker.go [hunk 1]
```diff
@@ -25,20 +25,20 @@ func main() {
 	}
 	// FIXME: Switch d and D ? (to be more sshd like)
 	flVersion := flag.Bool("v", false, "Print version information and quit")
-	flDaemon := flag.Bool("d", false, "Daemon mode")
-	flDebug := flag.Bool("D", false, "Debug mode")
+	flDaemon := flag.Bool("d", false, "Eanble daemon mode")
```
Comment (vieux) [inline docker/docker.go]: Typo "Eanble"

Comment (tianon) [inline docker/docker.go]: Doh!  Thanks.  "How embarrassing."  Fix pushed.

**Comment by vieux** [comment]
LGTM

**Pull request merged**